stream-log-android-file/src/main/kotlin/io/getstream/log/android/file/StreamLogFileService.kt

```diff
@@ -16,18 +16,57 @@
 
 package io.getstream.log.android.file
 
+import android.app.NotificationChannel
+import android.app.NotificationManager
 import android.app.Service
+import android.content.Context
 import android.content.Intent
+import android.os.Build
 import android.os.IBinder
+import androidx.annotation.RequiresApi
+import androidx.core.app.NotificationCompat
 
-private const val ACTION_SHARE = "io.getstream.logging.android.SHARE"
-private const val ACTION_CLEAR = "io.getstream.logging.android.CLEAR"
+private const val ACTION_SHARE = "io.getstream.log.android.SHARE"
+private const val ACTION_CLEAR = "io.getstream.log.android.CLEAR"
```

Comment:
The actions declared on AndroidManifests are different, so the current version `1.0.1` breaks this entire feature.